crates/autopilot/src/arguments.rs

```diff
@@ -139,8 +104,10 @@ pub struct Arguments {
 impl std::fmt::Display for Arguments {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         write!(f, "{}", self.shared)?;
+        write!(f, "{}", self.order_quoting)?;
```

Comment:
I've seen a few times that parameters are not getting logged with these manual `Display` implementations.

I *think* we should switch back to `derive(Debug)` with `derivative` for overriding the formatting methods to make these user errors not happen anymore.